site/docs/foundations/spacing.mdx

```diff
@@ -39,22 +39,51 @@ The Salt spacing system is structured around a base unit of `--salt-spacing-100`
 
+## Fixed spacing
+
+Fixed spacing offers tokens with values that stay the same across all densities. For instance, `--salt-spacing-100` is consistently 1px, and `--salt-spacing-200` is 2px, no matter the density. This provides a stable reference for design elements needing uniform spacing, ensuring consistency and simplicity within the design system.
+
+| Token                      | All densities (px) |
+| -------------------------- | ------------------ |
+| `--salt-spacing-fixed-100` | 1                  |
+| `--salt-spacing-fixed-200` | 2                  |
+| `--salt-spacing-fixed-300` | 3                  |
+| `--salt-spacing-fixed-400` | 4                  |
+| `--salt-spacing-fixed-500` | 5                  |
+| `--salt-spacing-fixed-600` | 6                  |
+| `--salt-spacing-fixed-700` | 7                  |
+| `--salt-spacing-fixed-800` | 8                  |
+| `--salt-spacing-fixed-900` | 9                  |
+
+Consider a numerical input with two buttons to its left, allowing users to step through values incrementally. Fixed spacing tokens define the distance between the input and the buttons, as well as the space between the two buttons. These spaces remain constant, ensuring precise control and uniformity. Meanwhile, the space between the numerical input and other components adjusts with density, providing flexibility and adaptability in the design.
+
+<ImageSwitcher
+  images={[
+    { src: "/img/foundations/spacing-fixed.png", alt: "" },
+    { src: "/img/foundations/spacing-fixed-annotated.png", alt: "" },
+  ]}
+  label="Show spacing"
+/>
+
 ## Hierarchical spacing
 
-Spacing is a powerful tool to emphasize certain elements while de-emphasizing others, guiding the user's journey through the content. Salt recommends using varied spacing (`--salt-spacing-300`, `--salt-spacing-200`, `--salt-spacing-100`) between elements to establish a visual hierarchy. The vertical separation of items such as headings, blocks of text, form elements and cards should follow a consistent pattern to create visual rhythm.
+Spacing is a key tool for emphasizing certain elements while downplaying others, guiding users through content. We suggests using varied spacing (`--salt-spacing-300`, `--salt-spacing-200, `--salt-spacing-100`) to establish a visual hierarchy. Consistent vertical separation of items like headings, text blocks, form elements, and cards creates a visual rhythm.
 
-Horizontal dividers are used to divide regions of a layout and should be spaced consistently with `salt-spacing-300` above and below sections. For more information on dividers, please refer to the [divider component page](/salt/components/divider).
+Horizontal dividers help segment layout regions and should be spaced consistently with `--salt-spacing-300` above and below sections. For more details on dividers, please refer to the divider component.
```

Comment:
Can we change to "please refer to Divider" and make Divider a link to the component?